crm_action/models/crm_action.py

```diff
@@ -48,30 +41,72 @@ def check_change(self):
         default=lambda self: self.env.user)
 
     def search_action_types(self):
-        return self.env['crm.action.type'].search(
-            [('is_active', '=', True)], order='priority')
+        return self.env['crm.action.type'].search([], order='priority')
 
     def default_action_type(self):
         action_types = self.search_action_types()
         return action_types and action_types[0].id or False
 
-    action_type = fields.Many2one(
+    action_type_id = fields.Many2one(
         'crm.action.type', string='Type', required=True,
         default=default_action_type)
 
     details = fields.Text('Details')
 
     state = fields.Selection(
         [
-            ('draft', 'Draft'),
+            ('draft', 'Todo'),
             ('done', 'Done'),
-        ], string='Status', required=True,
+        ], string='Status', required=True, readonly=True,
         default="draft")
 
     @api.multi
+    @api.depends('action_type_id.name', 'details')
+    def compute_display_name(self):
+        for action in self:
+            if action.details:
+                action.display_name = u'[%s] %s' % (
+                    action.action_type_id.name, action.details)
+            else:
+                action.display_name = u'[%s]' % action.action_type_id.name
+
+    display_name = fields.Char(
+        compute='compute_display_name', readonly=True, store=True)
+
+    @api.multi
     def button_confirm(self):
         self.write({'state': 'done'})
 
     @api.multi
     def button_set_to_draft(self):
         self.write({'state': 'draft'})
+
+    @api.model
+    def _send_email_reminder(self):
+        today = fields.Date.context_today(self)
+        actions = self.env['crm.action'].search([
+            ('state', '=', 'draft'), ('date', '=', today)])
+        logger.info(
+            'Preparing CRM actions email reminders '
+            '(%d actions found for today)',
+            len(actions))
+        user_company_actions = {}
+        # key = (user, company)
+        # value = list of crm.action records
+        if actions:
+            for action in actions:
+                user_company_actions.setdefault(
+                    (action.user_id, action.company_id), []).append(action)
```

Comment:
I don't think that what you propose is "more clear"... I tend to prefer my version :)